src/app/components/app-layout/app-layout.component.scss

```diff
@@ -18,7 +18,9 @@
  */
 
 @import "assets/styles/variables";
-
+.content-container{
+  min-height: calc(100vh - 324px);
```

Comment:
What adds up to 324? The footer and navigation combined? Seems like a magic number!

Reply:
Yes it's the nav and footer combined